Extensions/xinbox.js

```diff
@@ -304,10 +304,11 @@ XKit.extensions.xinbox = new Object({
 		var x_html = "<div id=\"xinbox-search-box\"><input type=\"text\" placeholder=\"Enter URL/text...\" id=\"xinbox-search-box-input\"></div>";
 		$("#xinbox_sidebar").before(x_html);
 
-		$("#xinbox_search_button").bind("click", function() {
```

Comment:
.on() and its shortcuts are the preferred method of handlers according to jQuery docs. Guess I didn't need to touch that thought, though.